src/main/ui/app/route/SearchPage.jsx

```diff
@@ -33,7 +33,8 @@ export default class SearchPage extends Component {
     doSearch() {
         const query = this.props.location.query.q;
         const type = this.props.location.query.type;
-        const searchKey = `type:${type ? type : "all"} query:${query ? query : ""}`;
+	const tag = this.props.location.query.tag;
```

Comment:
Formatting; this should have the same indentation as the lines before and after.